UnboundLib/Utils/LevelManager.cs

```diff
@@ -234,10 +226,7 @@ public static string[] GetLevelsInCategory(string category)
 
         public static string GetVisualName(string path)
         {
-            var visualName = path
-                .Replace(" ", "")
-                .Replace("Assets", "")
-                .Replace(".unity", "");
+            var visualName = path.Replace(" ", "").Replace("Assets", "").Replace(".unity", "");
```

Comment:
Repeated calls like this are most readable on separate lines. Recommend reverting